CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
```

Comment:
Please remove `Unreleased` and add the header for the next release version. I'm proving subject to this.